pkg/migrations/sql/preinstall/006-set_up_ha.sql

```diff
@@ -0,0 +1,146 @@
+CREATE TABLE SCHEMA_CATALOG.ha_locks
```

Comment:
I know this is from the design doc. But, I think changing this name to `ha_lease_state` is more descriptive and better understanding. Frankly, `ha_locks` does not describe anything.